packages/airnode-deployer/docker/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -e
```

Comment:
I ran into some errors (first I had the wrong folder setup and then the wrong `nodeVersion` in `config.json`, 0.8 vs. 0.7.1) and I think this line causes the container to exit immediately if there is an error, so I had to do the `docker logs...` to see what happened.

I suspect this is the intentional flow, but thought I'd note it here in case anyone else is wondering why the container might be suddenly exiting.

Reply:
Did you run docker in detached mode? We recommend that approach in https://docs.api3.org/airnode/v0.7/grp-providers/docker/client-image.html#checking-airnode-logs but I remember I disliked this.

The `set -e` is needed, because when the `node /app/src/cli/validate-config.js` fails, the bash script continues (with Airnode starting) which should not happen.

Reply:
Yep I ran it with detached.

> The set -e is needed, because when the node /app/src/cli/validate-config.js fails, the bash script continues (with Airnode starting) which should not happen.

This makes perfect sense